### DISCUSSION TITLE
Does surrounding a word with `` turns text inside into a code in outline ?

### DISCUSSION THREAD

**Discussion by bmm-alc** [outline/outline#3379] [General]
Hello

It does not work for me, I have to use the toolbar to do that each time.
it might be another bug with of US international layout I use with my keyboard as #3371  

version: 0.63.0-1

**Comment by tommoor** [comment]
Two more cases where this didn't correctly replace were fixed just yesterday:
https://github.com/outline/outline/issues/3375
